services/portal/values-idfprod.yaml

```diff
@@ -3,8 +3,8 @@ firefly:
   replicaCount: 4
 
   volumes:
-    firefly_workarea_nfs:
-      path: /share1/home/firefly/workarea
+    firefly_shared_workarea_nfs:
+      path: /share1/home/firefly/shared-workarea
```

Comment:
Here you're using Google Cloud resources to create a persistent NFS filesystem that's visible from all the pods, under that pathname, right?

Reply:
Correct.  I did that by hand, but we will want to roll it into our FS provisioning automation.